protocol/contracts/dao/Comptroller.sol

```diff
@@ -141,7 +130,7 @@ contract Comptroller is Setters {
 
     function mintToTreasury(uint256 amount) private {
         if (amount > 0) {
-            gold().mint(Constants.getTreasuryAddress(), amount);
+            gold().mint(getTreasuryAddress(), amount);
```

Comment:
why do lines 62 and 66 require `Constants.` but this line does not?

Reply:
It's a good point.  I think if I am going to change the lookup methods in place, I should do it as part of a larger refactor.  The reason I only changed the treasury address in this PR is that the change was concerning it.  I've reverted it though, lets leave it for a later task where the QA effort will be much higher.